pkg/agent/agent.go

```diff
@@ -673,13 +681,12 @@ func (i *Initializer) setupGatewayInterface() error {
 }
 
 func (i *Initializer) configureGatewayInterface(gatewayIface *interfacestore.InterfaceConfig) error {
-	var gwMAC net.HardwareAddr
 	var gwLinkIdx int
 	var err error
 	// Host link might not be queried at once after creating OVS internal port; retry max 5 times with 1s
 	// delay each time to ensure the link is ready.
 	for retry := 0; retry < maxRetryForHostLink; retry++ {
-		gwMAC, gwLinkIdx, err = util.SetLinkUp(i.hostGateway)
+		_, gwLinkIdx, err = setLinkUp(i.hostGateway)
```

Comment:
I think the first argument could be removed as it's no longer used by any caller

Reply:
Removed the first return value in function `setLinkUp`